RELEASES.md

```diff
@@ -1,8 +1,22 @@
-# Release Process
-This document explains the process to release new version and to address patch release.
+# Releases
+This document explains the process to release new versions.
+
+## Versioning
+Versions are expressed as `vX.Y.Z` as well as Semantic Versioning;
+
+- `X` is the major version
+- `Y` is the minor version
+- `Z` is the patch version
+
+## Release Cycle
+- Major releases: arbitrary timing
+- Minor releases: roughly every 2 months
+- Patch releases: roughly every 2-3 weeks
+
+Release candidates(vX.X.X-alpha) for minor versions are available for 2-4 weeks before the release.
 
 ## Major release
-This refers to the release of new features.
+This refers to the release of breaking changes.
```

Comment:
```suggestion
This refers to the release of new features or breaking changes.
```
If only breaking changes, then this would rarely happen since we don't expect breaking change.

Reply:
I think PipeCD should only once release major version at some point. That way,  we can explain a major version is `This refers to the release of breaking changes (PipeCD committed to backward compatibility, so a next breaking changes will not happen)`.

Reply:
In such case, all new features also only be released as patch, that sounds not really nice 🤔

Reply:
I focus on major release.  I think all new features is released as minor, if PipeCD secure backward compatibility or do these feature have no backward compatibility you mention to ?

Reply:
If we increment major versions(`X`), minor versions(`Y`) always change, too. So I guess the explanation of major releases can simply refer to "breaking changes".